### DISCUSSION TITLE
blog/non-mi-sazia-niente

### DISCUSSION THREAD

**Discussion by giscus** [peppocola/pompelmo#13] [Q&A]
# blog/non-mi-sazia-niente

Farewell Pompelmo

https://www.pompelmo.xyz/blog/non-mi-sazia-niente

**Comment by pippokill** [comment]
La mia domanda è: perché corri per raggiungere un unico grande traguardo che colmi quella fame? Secondo me devi modificare le premesse :) Non credo che la vita sia una corsa contro l'unico mega traguardo che ti gratifichi di tutto. Viviti ogni passo, ogni piccolo traguardo e fallimento, perché non solo di vittorie si può campare.

**Reply by peppocola** [reply]
non è facile uscire da questo meccanismo. e razionalmente ti dò pienamente ragione. ma io ci sono dentro. non è una cosa che ha senso. è quello che mi muove da sempre e cambiare questa cosa è molto difficile
